src/error/specification

```diff
@@ -1209,3 +1209,9 @@ severity:error
 ingroup:plugin
 module:typechecker
 macro:PLUGIN_TYPECHECKER_SET
+
+number:194
+description:The given path does not meet the correct permissions
```

Comment:
We should try to reuse/reduce the errors and make a nice classification system. Hopefully you can contribute to improve the error system.